frontend/e2e-tests/dom-to-db-tests/data-entry.e2e.ts

```diff
@@ -297,10 +297,10 @@ test.describe("full data entry flow", () => {
     await candidatesListPage_1.next.click();
 
     const checkAndSavePage = new CheckAndSavePage(page);
-    await checkAndSavePage.fieldset.waitFor();
+    await checkAndSavePage.fieldset.isVisible();
```

Comment:
```suggestion
    await expect(checkAndSavePage.fieldset).toBeVisible();
```
shouldn't this one also have an added expect?